web-app/src/app/tabs/mppt/mppt.component.css

```diff
@@ -0,0 +1,127 @@
+.header {
+	display: flex;
+	justify-content: space-between;
+	font-family: Arial, sans-serif;
+	font-size: 1.2rem;
+	background-color: transparent;
+	color: #D5FEFF;
+	padding-bottom: 0.2em;
+	padding-top: .2em;
+	margin-top: 0.1em;
+	margin-bottom: 0.1em;
+	width: auto;
+}
+.header-title-left {
+	float: left;
+	width: 45%;
+	margin-left: 5%;
+}
+.title-left {
+	float: left;
+	width: 47.5%;
+	margin-right: 2.5%;
+}
+.title-right {
+	float: left;
+	width: 47.5%;
+	margin-left: 2.5%;
+}
+
+.text {
+	text-align: center;
+}
+.aliveCircle {
+	float: left;
+}
+
+.circle {
+	float: left;
+	width: 15px;
+	height: 15px;
+}
+
+.circleItems {
+	width: 15px;
+	height: 15px;
+}
+
+.button {
+	float: left;
+	width: 10px;
+	height: 10px;
+}
+
+.ifs {
+	width: 10px;
+}
+
+.spacing {
+	width: 40%;
+	margin-left: 20%;
+}
+
+.centering {
+	width: 40%;
+	margin-left: 5%;
+	margin-right: 5%; 
+}
+
+
+.reduce {
+	width: 20%;
+}
+
+.subheader {
+	font-family: Arial, sans-serif;
+	font-size: 0.8rem;
+	background-color: grey;
+	color: rgba(255, 255, 255, 0.75);
+	padding-bottom: 0.2em;
+	padding-top: .2em;
+	border-radius: 5px;
+	text-align: center;
+	margin-bottom: 15px;
+}
+.subheader > h3 {
+	font-family: Arial, sans-serif;
+	font-size: .8em;
+	background-color: #7a7676;
+	color: rgba(255, 255, 255, 0.75);
+	border-radius: 25px;
+}
+
+.container-row {
+	padding-top: .2em;
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+}
+.container-column {
+	display: inline-flex;
+	flex-direction: column;
+	justify-content: center;
+}
+.item {
```

Comment:
`.item` and `.data` have the exact same stylesheet, so why have it any different?